<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpritePackerWidget</class>
 <widget class="QWidget" name="SpritePackerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="m_splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="m_layout">
       <item>
        <widget class="QTreeView" name="m_tree_sprites">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btn_add_sprites">
         <property name="text">
          <string>Add Sprites</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <layout class="QVBoxLayout" name="m_layout_2">
       <item>
        <widget class="GraphicsView" name="m_preview">
         <property name="dragMode">
          <enum>QGraphicsView::DragMode::RubberBandDrag</enum>
         </property>
         <property name="transformationAnchor">
          <enum>QGraphicsView::ViewportAnchor::AnchorUnderMouse</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignRight">
        <widget class="ZoomWidget" name="m_zoom_widget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_panel_properties" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>380</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="m_label_max_width">
        <property name="text">
         <string>Max width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="m_spin_max_width">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>128</number>
        </property>
        <property name="maximum">
         <number>65536</number>
        </property>
        <property name="value">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_label_max_height">
        <property name="text">
         <string>Max height</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="m_spin_max_height">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>128</number>
        </property>
        <property name="maximum">
         <number>65536</number>
        </property>
        <property name="value">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Packer</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="m_label_algorithm">
           <property name="text">
            <string>Algorithm</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="m_combo_algorithm">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="m_label_mrb_heuristic">
           <property name="text">
            <string>Choice heuristic</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="m_combo_mrb_heuristic">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="m_checkbox_mrb_allow_flip">
           <property name="text">
            <string>Allow flip</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="m_label_skyline_heuristic">
           <property name="text">
            <string>Choice heuristic</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="m_combo_skyline_heuristic">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="m_checkbox_skyline_use_waste_map">
           <property name="text">
            <string>Use waste map</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="m_label_guillotine_choice_heuristic">
           <property name="text">
            <string>Choice heuristic</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="m_combo_guillotine_choice_heuristic">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="m_label_guillotine_split_heuristic">
           <property name="text">
            <string>Split heuristic</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="m_combo_guillotine_split_heuristic">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QCheckBox" name="m_checkbox_guillotine_allow_merge">
           <property name="text">
            <string>Allow merge</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="m_label_shelf_choice_heuristic">
           <property name="text">
            <string>Choice heuristic</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QComboBox" name="m_combo_shelf_choice_heuristic">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QCheckBox" name="m_checkbox_shelf_use_waste_map">
           <property name="text">
            <string>Use waste map</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomWidget</class>
   <extends>QWidget</extends>
   <header location="global">Sol2dTexturePackerGui/ZoomWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">Sol2dTexturePackerGui/GraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_btn_add_sprites</tabstop>
  <tabstop>m_spin_max_width</tabstop>
  <tabstop>m_spin_max_height</tabstop>
  <tabstop>m_combo_algorithm</tabstop>
  <tabstop>m_combo_mrb_heuristic</tabstop>
  <tabstop>m_checkbox_mrb_allow_flip</tabstop>
  <tabstop>m_combo_skyline_heuristic</tabstop>
  <tabstop>m_checkbox_skyline_use_waste_map</tabstop>
  <tabstop>m_combo_guillotine_choice_heuristic</tabstop>
  <tabstop>m_combo_guillotine_split_heuristic</tabstop>
  <tabstop>m_checkbox_guillotine_allow_merge</tabstop>
  <tabstop>m_combo_shelf_choice_heuristic</tabstop>
  <tabstop>m_checkbox_shelf_use_waste_map</tabstop>
  <tabstop>m_tree_sprites</tabstop>
  <tabstop>m_preview</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
